drivers/misc/Kconfig

```diff
@@ -81,6 +81,10 @@ config BLK_RPMSG_SERVER
 	default n
 	depends on RPMSG
 
+config GOLDFISH_PIPE
+	bool "Goldfish Pipe Support"
+	default n
+
```

Comment:
Please include help with explanation about that is this feature and when user need to select it.